apps/client/src/app/modules/list/list-details-panel/list-details-panel.component.ts

```diff
@@ -180,22 +194,34 @@ export class ListDetailsPanelComponent implements OnChanges, OnInit {
       this.hasNavigationMap = this.getZoneBreakdownPathRows(this.zoneBreakdown).length > 0;
     }
     this.hasTrades = this.displayRow.rows.reduce((hasTrades, row) => {
-      return (this.getData(row, DataType.TRADE_SOURCES).length > 0) || (this.getData(row, DataType.VENDORS).length > 0) || hasTrades;
+      return ListRowSerializationHelper.getData(row, DataType.TRADE_SOURCES).length > 0
```

Comment:
Why are you changing these calls to a static method in the serialization helper that just calls the getDataSource method directly?

Reply:
not realizing this could be entirely replaced with `getItemSource<Type>` and since I had moved `getData` I had the old code reference the new location. this should get reverted or call `getItemSource` directly. not sure why it wasn't from the getgo.

Reply:
Using `getItemSource` is the best way to do it, I used to use a custom method inside the component when I first implemented this logic, then I changed it but didn't migrate this part.